localstack/utils/container_utils/docker_cmd_client.py

```diff
@@ -831,3 +832,13 @@ def _check_and_raise_no_such_container_error(
         process_stdout_lower = to_str(error.stdout).lower()
         if any(msg.lower() in process_stdout_lower for msg in error_messages):
             raise NoSuchContainer(container_name_or_id, stdout=error.stdout, stderr=error.stderr)
+
+    def _transform_container_labels(self, labels: str) -> Dict[str, str]:
```

Comment:
How stable is this format do you think? Is there any chance the Docker client might add any whitespace, for example?

Reply:
This is only used on the return values of `docker ps` with the format set to `json`. I think it should be rather stable, as a change there could break a lot of automated tools.